skimage/io/hdr.py

```diff
@@ -0,0 +1,291 @@
+import numpy as np
+
+
+def getRadiance(ims, exp, radianceMap):
```

Comment:
please use PEP8 standard naming conventions (i.e. `get_radiance` and `radiance_map`).